config.reek

```diff
@@ -2,4 +2,15 @@ IrresponsibleModule:
   enabled: false
 
 NestedIterators:
-  enabled: false
\ No newline at end of file
+  enabled: false
+
+"app/controllers":
+  TooManyStatements:
+    max_statements: 10
```

Comment:
I believe we can fix the controller in a way that we don't need to write an exception here.

Reply:
I did tried, but couldn't found a good solution here. The controller has 6 statements and that's it. 
Moving the method to service layer would probably fix the case and overcome default setting of 5. But does it really worth it?